op-monitorism/transaction_monitor/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "watch_configs": [
+    {
+      "address": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
+      "name": "Optimism L1 Bridge",
+      "allow_list": [
+        "0x4200000000000000000000000000000000000010",
+        "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
+        "0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1"
+      ],
+      "thresholds": {
+        "0x4200000000000000000000000000000000000010": "100000000000000000000",
+        "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1": "1000000000000000000",
+        "0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1": "100000000000000000000"
+      }
+    }
+  ]
+}
```

Comment:
For arbitrary configurations, I would prefer we use yaml. We have here an example where we use it: https://github.com/ethereum-optimism/monitorism/blob/main/op-monitorism/global_events/rules/rules_mainnet_L1/rules_TEMPLATE_COPY_PASTE.yaml

With yaml we have a human readable, machine readable and comment friendly format.